main.nf

```diff
@@ -641,6 +645,7 @@ if(params.aligner == 'hisat2'){
                    --met-stderr \\
                    --new-summary \\
                    --summary-file ${prefix}.hisat2_summary.txt \\
+                   $RG \\
```

Comment:
When we have options that are likely to be blank, I prefer for them not to be on their own line. Although it shouldn't affect things, it looks strange to see newlines in command scripts. Should be fine to just append the variable to the above line.